framework/src/wifi_manager/wifi_manager_api.c

```diff
@@ -100,7 +100,7 @@ wifi_manager_result_e wifi_manager_init(wifi_manager_cb_s *wmcb)
 		return WIFI_MANAGER_INVALID_ARGS;
 	}
 
-	wifimgr_msg_s msg = {EVT_INIT_CMD, (void *)wmcb, NULL};
+	wifimgr_msg_s msg = {EVT_INIT_CMD, WIFI_MANAGER_FAIL, (void *)wmcb, NULL};
```

Comment:
The second parameter is the result whether the Wi-Fi manager successfully performs the request, which will be delivered asynchronously.
Then, what is the return value of this function? Is it just a result whether the request is successfully delivered to the Wi-Fi manager?
Then, we need to clarify this difference to improve API usability.

Reply:
The return value of wifimgr_post_message is for checking that the msg is passed to wifi msg handler thread well. however this return value and second parameter are not separated well. so I fixed it.